py/dml/dmlparse.py

```diff
@@ -1144,6 +1144,9 @@ def sizespec_empty(t):
 @prod
 def offsetspec(t):
     'offsetspec : AT expression'
+    if (t[2].kind == 'undefined' and site(t).dml_version() == (1, 2)
```

Comment:
I'd reorder the conjunction with the version check first to highlight that this is dying code.

Reply:
> Externally reported, so deserves a releasenote.

Which `RELEASENOTES` should that be in?